Index: out/production/pogl/.idea/.name
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Main.java
===================================================================
diff --git a/out/production/pogl/.idea/.name b/out/production/pogl/.idea/.name
--- a/out/production/pogl/.idea/.name	(revision 6799b717a98b3e8325a2d6a9ab0c786108559b79)
+++ b/out/production/pogl/.idea/.name	(date 1743337245322)
@@ -1,1 +1,1 @@
-Main.java
\ No newline at end of file
+Zone.java
\ No newline at end of file
